This is Solaris specific patch to place the "pid" and "socket"
file/socket to /var/run
Will not promote to community.

--- a/configure.ac	2017-07-10 03:14:59.131370037 +0000
+++ b/configure.ac	2017-11-07 04:31:12.840517817 +0000
@@ -958,7 +969,7 @@
 #
 # Avahi runtime dir
 #
-avahi_runtime_dir="/run"
+avahi_runtime_dir="/var/run"
 avahi_socket="${avahi_runtime_dir}/avahi-daemon/socket"
 AC_SUBST(avahi_runtime_dir)
 AC_SUBST(avahi_socket)
